package.json

```diff
@@ -50,6 +50,7 @@
     "testMain": "export NODE_OPTIONS=--max-old-space-size=2048; karma start karma.conf.js",
     "testModules": "lerna run test --ignore=cbioportal-frontend --stream",
     "test:watch": "yarn run test -- --watch",
+    "testMain:watch": "yarn run testMain -- --watch",
```

Comment:
@onursumer is there an existing way to run only the main tests?

Reply:
@alisman you mean to watch only the main tests, right? Seems like we don't. We need to add `testMain:watch`.

Ideally main package should be under some directory like `webapps/cbioportal`. We could just do `yarn workspace cbioportal test:watch` in that case. For the current setup though we still need workarounds like `testMain` and `testMain: watch` because we use root `package.json` for both managing the main package and the monorepo setup.